app/views/admin/student.erb

```diff
@@ -13,13 +13,14 @@ Back
   </thead>
 
   <tbody>
-    <% @assignments.each do |a| %>
+    <%= @assignments.each do |a| %>
```

Comment:
`<%= @assignments.each do |a| %>`

Lose the `=` here and see if that makes a difference.